docs/reporting-services/report-data/report-data-pane.md

```diff
@@ -27,7 +27,7 @@ manager: "kfile"
 ms.workload: "Inactive"
 ---
 # Report Data Pane
-  Use the **Report Data** pane to view the currently defined parameters, data sources, datasets, field collections, and images in your report. The Report Dane displays a hierarchical view of the items that represent data in your report. The top level nodes represent built-in fields, parameters, images, and data source references. Expand each node to view the data items. For example, when you expand a data source node, the datasets defined for that data source appear. When you expand a dataset, its field collection appears. Drag items to the report design surface to link data with report items on the report page.  
+  Use the **Report Data** pane to view the currently defined parameters, data sources, datasets, field collections, and images in your report. The Report Data pane displays a hierarchical view of the items that represent data in your report. The top level nodes represent built-in fields, parameters, images, and data source references. Expand each node to view the data items. For example, when you expand a data source node, the datasets defined for that data source appear. When you expand a dataset, its field collection appears. Drag items to the report design surface to link data with report items on the report page.  

```

Comment:
just a small suggestion.  I would change "top level" to "top-level" maybe?

Reply:
I see why the addition of a dash, to replace the space in "top level", might be suggested.  But we should lean toward avoid unnecessary uses of dashes, and I feel that the text is clear and at least equally correct in this case without adding the dash.
Thanks for caring enough to chime in!  Seriously thanks.  :-) 
GeneMi at MS (MightyPen)

Reply:
@MightyPen I approve. 👍